docs/SDK/Explanation/authentication/sessionSigs/getSessionSigs.md

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_position: 2
+---
+
+# Generating SessionSigs
+
+You can use any wallet or auth method to generate session signatures with the `getSessionSigs()` function from the Lit SDK. This function generates a session keypair and uses a callback function that signs the generated session key to create an `AuthSig` that is scoped to specific capabilities.
+
```

Comment:
Since below we generate a wallet and hand role the `SIWE`  we should say they can use the `checkAndSignAuthMessage` function in the sdk if they are in a browser context.